ffi/src/handle.rs

```diff
@@ -72,6 +72,9 @@ macro_rules! def_ffi_fn {
             },
         }
     };
+    ( $prefix:literal $($fn_name:ident: {$($other:ty),+ $(,)?}),+ $(,)?) => {
+        $( $crate::def_ffi_fns! {$prefix $fn_name: $( $other ),+ } )+
+    };
```

Comment:
```suggestion
    ( $($prefix:literal)? $($fn_name:ident: {$($other:ty),+ $(,)?}),+ $(,)?) => {
        $( $crate::def_ffi_fns! {$prefix $fn_name: $( $other ),+ } )+
    };
```

I wanted to make `prefix` optional like this but I just don't know how to

Reply:
Most straight forward approach will be to just add one more rule without prefix.

Reply:
haha, that's it. Thank you

Reply:
> Most straight forward approach will be to just add one more rule without prefix.

Actually that would also be faster, given that the rule captures are slower than rule matching.